Vagrantfile

```diff
@@ -0,0 +1,28 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  config.vm.box = 'ubuntu/xenial64'
+  config.vm.define "prescript"
```

Comment:
Mmmm, I forgot to change the name of this machine in the script. Let's change it to "atlas" or the like.